fs_local.go

```diff
@@ -114,25 +114,36 @@ func (fs LocalFileSystem) ReadDir(ctx context.Context, name string, recursive bo
 	return l, errFromOS(err)
 }
 
-func (fs LocalFileSystem) Create(ctx context.Context, name string, body io.ReadCloser) (*FileInfo, error) {
+func (fs LocalFileSystem) Create(ctx context.Context, name string, body io.ReadCloser) (*FileInfo, bool, error) {
 	p, err := fs.localPath(name)
 	if err != nil {
-		return nil, err
+		return nil, false, err
+	}
+	created := false
+	fi, _ := fs.Stat(ctx, name)
```

Comment:
I think it should be `p` instead of `name`.

Oops, it sounds like I didn't catch the issue when reviewing the previous PR: the `Stat` call at the end of the function has the same issue.

Nit: could be simplified with something like:

```go
created := fi == nil
``

Reply:
Oh yes, indeed!